core/domain/voiceover_domain.py

```diff
@@ -185,25 +186,28 @@ def add_voiceover(
         voiceover_type: feconf.VoiceoverType,
         voiceover: state_domain.Voiceover
     ) -> None:
-        """Adds voiceover to the entity voiceover instance."""
-        if not isinstance(voiceover_type, feconf.VoiceoverType):
-            raise utils.ValidationError(
-                'voiceover type must be VoiceoverType, received %s' %
-                voiceover_type)
-        voiceover.validate()
+        """Adds voiceover to the entity voiceover instance.
+
+        Args:
+            content_id: str. The ID of the content.
+            voiceover_type: VoiceoverType. The voiceover type of the given
+                voiceover.
+            voiceover: Voiceover. An instance of Voiceover class.
```

Comment:
Docstrings should not describe just the type; those docstrings are useless.

In this case you want to describe the voiceover and how the function will use it, e.g. "The voiceover to be added."